the-bureau/index.html

```diff
@@ -14,6 +14,39 @@
     <link rel="stylesheet" href="/static/vendor/slick-carousel/slick.css">
 {%- endblock %}
 
+{% block hero -%}
+    <section class="content_hero hero hero__bureau">
+        {% import "macros.html" as macros %}
+        {% call macros.youtube({
+            'video_id' : 'en0Iq8II4fA',
+            'image_res' : 'max' 
+            })
+        %}
```

Comment:
Suggest modifying the line to follow this format:

`{% from "macros.html" import youtube as youtube %}`

Which means you can simplify the call to just:

```
{{ youtube(
    'video_id' : 'en0Iq8II4fA',
    'image_res' : 'max'
}) }}

(See the implementation of the `share` macro in `article.html` for an example.)